### PR TITLE
Improve the anitya processor for project.flag.set messages

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

**Comment by sayanchowdhury** [comment]
Looks good to me :+1:

**Comment by pypingou** [comment]
Looks good but travis doesn't seem happy :)

File: fedmsg_meta_fedora_infrastructure/anitya.py [hunk 1]
```diff
@@ -245,7 +245,7 @@ def objects(self, msg, **config):
         elif 'distro.remove' in msg['topic']:
             distro = msg['msg']['distro']['name']
             return set(['distros/%s' % distro])
-        elif.msg['topic'].endswith('project.flag.set'):
+        elif msg['topic'].endswith('project.flag.set'):
```
Comment (sayanchowdhury) [inline fedmsg_meta_fedora_infrastructure/anitya.py]: Oh! My eyes did not catch that :\

**Comment by sayanchowdhury** [comment]
:+1:

**Comment by pypingou** [comment]
Thanks

**Pull request merged**